applications.rst

```diff
@@ -2,36 +2,94 @@
 Applications
 ============
 
-.. toctree::
-   :titlesonly:
-
-   accounting
-   expense/expense
+.. ORDER (new apps / sections included):
+   general
+   discuss
    crm
    sales
-   website
-   live_chat
-   ecommerce
    purchase
+   rental
+   accounting
+   consolidation
+   subscription
+   project
+   timesheets
+   planning
+   bi
+   documents
+   fsm
+   point_of_sale
    inventory
    manufacturing
    quality
-   point_of_sale
-   discuss
+   barcode
+   iot
+   website
+   ecommerce
+   elearning
    helpdesk
+   live_chat
+   email_marketing
+   social_marketing
+   sms_marketing
+   marketing_automation
+   events
+   survey
+   employees
+   recruitment
+   referrals
+   leaves
+   expenses
+   appraisals
+   fleet
+   approvals
+   lunch
+   studio
+   mobile
+   misc
+
+.. toctree::
+   :titlesonly:
+
+   general
+   discuss
+   crm
+   sales
+   purchase
+
+   accounting
+
+
    project
-   fsm
-   planning
    timesheets
+   planning
+
+
+   fsm
+   point_of_sale
+   inventory
+   manufacturing
+   quality
+
    iot
-   mobile/firebase
-   events
+   website
+   ecommerce
+
+   helpdesk
+   live_chat
    email_marketing
    social_marketing
    sms_marketing
    marketing_automation
+   events
    survey
 
-   general
 
-.. Leave general on the last line !
\ No newline at end of file
+
+
+   expenses
+
+
+
+
```

Comment:
*With Gordon Ramsay's voice :*
There is so much space here I though it was a parking orbit !

Seriously, I don't think that leaving blank lines as placeholders for future doc is a good idea. In the end, people will just add a new line for their doc and keep the other blank lines.